Tests/XCUITests/HistoryTests.swift

```diff
@@ -38,45 +39,39 @@ class HistoryTests: BaseTestCase {
     }
 
     func testEmptyHistoryListFirstTime() {
-        navigator.performAction(Action.CloseURLBarOpen)
-        navigator.nowAt(NewTabScreen)
         // Go to History List from Top Sites and check it is empty
         navigator.goto(LibraryPanel_History)
         waitForExistence(app.tables.cells[HistoryPanelA11y.recentlyClosedCell])
-        XCTAssertTrue(app.tables.cells[HistoryPanelA11y.recentlyClosedCell].exists)
+        XCTAssertTrue(app.tables.cells[HistoryPanelA11y.recentlyClosedCell].staticTexts["Recently Closed"].exists)
+        XCTAssertTrue(app.tables[HistoryPanelA11y.tableView].staticTexts[emptyRecentlyClosedMesg].exists)
     }
 
     func testOpenSyncDevices() {
-        navigator.performAction(Action.CloseURLBarOpen)
-        navigator.nowAt(NewTabScreen)
+        // Firefox sync page should be available
         navigator.goto(TabTray)
         navigator.performAction(Action.ToggleSyncMode)
         waitForExistence(app.tables.cells.staticTexts["Firefox Sync"])
-        XCTAssertTrue(app.tables.buttons["Sign in to Sync"].exists, "Sing in button does not appear")
+        XCTAssertTrue(app.tables.buttons["Sync and Save Data"].exists, "Sign in button does not appear")
     }
 
     func testClearHistoryFromSettings() {
-        navigator.performAction(Action.CloseURLBarOpen)
-        navigator.nowAt(NewTabScreen)
         // Browse to have an item in history list
         navigator.goto(LibraryPanel_History)
-        waitForExistence(app.tables.cells[HistoryPanelA11y.recentlyClosedCell], timeout: 5)
+        waitForExistence(app.tables.cells[HistoryPanelA11y.recentlyClosedCell])
         XCTAssertTrue(app.tables.cells.staticTexts[webpage["label"]!].exists)
+        XCTAssertFalse(app.tables[HistoryPanelA11y.tableView].staticTexts[emptyRecentlyClosedMesg].exists)
 
         // Go to Clear Data
-        navigator.goto(NewTabScreen)
```

Comment:
Removing these lines and the following ones make the test fail for me, it is failing on Bitrise too... is this working for you?

Reply:
The tests are passing locally on my end before I committed the change. 🫤